src/java/org/apache/cassandra/metrics/KeyspaceMetrics.java

```diff
@@ -275,6 +279,10 @@ public KeyspaceMetrics(final Keyspace ks)
         rowIndexSizeAborts = createKeyspaceMeter("RowIndexSizeAborts");
         rowIndexSize = createKeyspaceHistogram("RowIndexSize", false);
 
+        tooManyIndexReadSSTablesWarnings = createKeyspaceMeter("TooManyIndexReadSSTablesWarnings");
```

Comment:
Does anything update these 3 metrics?

Reply:
We might also consider just moving these to `PerQueryMetrics`, but I'd be fine w/ keeping them here if we name appropriately. My proposal would be `TooManySSTableIndexesQueried[Warnings|Aborts]`.

@mike-tr-adamson WDYT?

Reply:
@smiklosovic two of them are updated in `CoordinatorWarnings.done()`